sessions/react-state-1/emoji-checker_solution/src/App.js

```diff
@@ -3,11 +3,11 @@ import "./styles.css";
 
 export default function App() {
   const [code, setCode] = useState("");
-  console.log(code);
+
   const validCode = "🐡🐠🐋";
 
-  function handleCode(emoji) {
-    setCode(code + emoji);
+  function handleClick(event) {
+    setCode(code + event.target.textContent);
```

Comment:
This could be a dangerous pattern, as users are abled to change the text contents in their browser, which could allow a user to feed textContent into your app, that differs from what you expect.

Reply:
This is how it is described in the challenge

```
- Write a `handleClick` function that updates the state variable according to which emoji button was clicked.
- Hint: `event.target.textContent` (use `code + event.target.textContent` to append the new emoji)
```